swim/member.go

```diff
@@ -61,6 +61,18 @@ func (m Member) incarnation() int64 {
 	return m.Incarnation
 }
 
+func (m *Member) populateFromChange(c *Change) {
+	m.Address = c.Address
+	m.Incarnation = c.Incarnation
+	m.Status = c.Status
+}
+
+// checksumString fills a buffer that is passed with the contents that this node
+// needs to add to the checksum string.
+func (m Member) checksumString(b *bytes.Buffer) {
+	fmt.Fprintf(b, "%s%s%v", m.Address, m.Status, m.Incarnation)
```

Comment:
I like the idea. Both `Address` and `Status` could written directly to the `Buffer` as it has a `WriteString` method (`io.Writer` only accepts `byte[]`). But `Incarnation` is an `int64` which can not be written directly. But we could use `strconv.Itoa()` to translate the number into a string.